docs/user/supported_devices.rst

```diff
@@ -261,6 +261,9 @@ lantiq-xway
 * AVM
 
   - FRITZ!Box 7312 [#avmflash]_
+  - FRITZ!Box 7320 [#avmflash]_ [#lan_as_wan]_
+  - FRITZ!Box 7330 [#avmflash]_ [#lan_as_wan]_
+  - FRITZ!Box 7330 SL [#avmflash]_ [#lan_as_wan]_
```

Comment:
Have you tested AND verified it with every distinct model? I'm asking as the kernel might run on every device, but GPIO assignment / flags might differ between models.

Reply:
OpenWRT is for the 7320 and my 7330SL works fine with it, every button and led works so i guess the 7320 would work fine also. The 7330 has only POTS extra but is the same as the 7330SL. 
According to some threads in the IP Phone Forum it is even possible to mix the Stock firmwares for the 3 models with some hacks.
But if you want i can test with an 7320 to be sure, i just bid on it on ebay for cheap ;).

Reply:
My 7320 arrived today and works flawlessly with the image.